wllvm/filetype.py

```diff
@@ -34,7 +35,7 @@ def getFileType(cls, fileName):
         Maybe we should use python-magic instead?
         """
         retval = None
-        fileP = Popen(['file', os.path.realpath(fileName)], stdout=PIPE)
+        fileP = Popen(['/usr/bin/file', os.path.realpath(fileName)], stdout=PIPE)
```

Comment:
so in Android build system, host tool can not be used as PATH var is reset. let me add a check here for /usr/bin/file or /usr/local/bin/file

Reply:
So I'm not going to merge in hard coded paths like this. If you look at the code base you will notice that most tools we use (via Popen) have the feature that you can specify the full path via an environment variable.  If it was well documented I would include this approach. But not a hardcoded path hack.